arch/arm/boot/dts/overlays/Makefile

```diff
@@ -104,6 +104,7 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	mpu6050.dtbo \
 	mz61581.dtbo \
 	ov5647.dtbo \
+	irs1125.dtbo \
```

Comment:
Sorry, you'll get a request from Phil to keep all of these in alphabetical order.

Reply:
There should also be an entry in README to describe what the overlay does.

Reply:
okay, I changed that. Having seen the entires in README, I changed my dtparams to contain i2c_pins_0_1 and ic2_pins_28_29 just like imx219 and ov5647